landingai/vision_pipeline.py

```diff
@@ -204,6 +210,49 @@ def save_image(self, filename_prefix: str, image_src: str = "") -> "FrameSet":
             c += 1
         return self
 
+    def save_video(
+        self,
+        video_file_path: str,
+        video_fps: Union[int, None] = None,
+        video_length_sec: Union[int, None] = None,
+        image_src: str = "",
+    ) -> "FrameSet":
+        """Save the FrameSet as an mp4 video file
+
+        Parameters
+        ----------
+        video_file_path : str
+            Path and filename with extension of the video file
+        video_fps : Union[int, None], optional
+            Either the frames per second or the video length should be provided to assemble the video
+        video_length_sec : Union[int, None], optional
+            Either the frames per second or the video length should be provided to assemble the video
```

Comment:
please also document the behavior when `video_fps` or `video_length_sec` is none

Reply:
Done